models/demos/t3000/falcon40b/demo/demo.py

```diff
@@ -563,13 +590,28 @@ def run_falcon_demo_kv(
     logger.info(f"decode inference time: {round(measurements['inference_decode'], 5)} s")
     logger.info(f"total inference time: {round(measurements['inference_total'], 5)} s")
     if not prefill_on_host:
-        logger.info(f"inference throughput prefill: {round(measurements['inference_throughput_prefill'], 5)} users/s")
         logger.info(
-            f"inference throughput prefill | seq_len={prefill_ids.shape[1]}: {round(measurements['inference_throughput_prefill']*prefill_ids.shape[1], 5)} tok/s"
+            f"inference throughput prefill: {round(measurements['inference_user_throughput_prefill'], 5)} users/s"
         )
-    logger.info(f"inference throughput decode: {round(measurements['inference_throughput_decode'], 5)} tok/s")
-    logger.info(
-        f"inference throughput decode (per user): {round(measurements['inference_throughput_decode']/batch_size, 5)} tok/s/user"
+        logger.info(
+            f"inference throughput prefill | seq_len={prefill_ids.shape[1]}: {round(measurements['inference_user_throughput_prefill']*prefill_ids.shape[1], 5)} tok/s"
+        )
+    logger.info(f"inference throughput decode: {round(measurements['decode_t/s'], 5)} tok/s")
+    logger.info(f"inference throughput decode (per user): {round(measurements['decode_t/s']/batch_size, 5)} tok/s/user")
+    profiler.end("run")
```

Comment:
Could you add the logging for time to first token and prefill+decode throughput as well?
`logger.info(f"time to first token (prefill, 1st user): {round(measurements['prefill_time_to_token'], 5)} s")`
`logger.info(f"inference throughput prefill+decode (per user): {round(measurements['prefill_decode_t/s/u'], 5)} tok/s/user")`